test/api/buffer/import/import/001-ok/request.json

```diff
@@ -0,0 +1,8 @@
+{
+  "path": "/v1/import/%%TEST_KBC_PROJECT_ID%%/receiver-1/#/2wsMsoyPA6YsDmnf7GJvqyxlDU6kPX7WWFu1pOMz1O7ybPVI",
```

Comment:
@michaljurecko Tak mám za sebou pár hodin debugování a vypadá to že Resty stripne ten anchor se secretem z url. 😕 A nemůžu tam najít žádnou metodu, kterou bych to do toho requestu injectnul ani že by to v jejich repu někdy někdo řešil.
![CleanShot 2022-11-23 at 20 32 32@2x](https://user-images.githubusercontent.com/215660/203632199-bb9380f2-7179-403d-8638-1de4144b6690.jpg)

Reply:
:slightly_frowning_face: resty by sme sa mali asi postupne zbavit, ... ked sa ti chce, tak mozes skusit pouzit nasho `go-client` v E2E testoch, uplne rovnako ako buildujeme requesty pre nase APIs.

Reply:
toto si asi videl https://stackoverflow.com/a/45537161

Reply:
moze to mazat aj server, tazko povedat, ... mozem na to mrknut rano

Reply:
ze to nie je v logoch je vlastne dobre a ciel, pre ktory to tak produkt chcel, ... ked sa loguje `path`, tak `fragment` sa nezvykne logovat.

Reply:
(osobne by som si dumpol raw `*http.Request` v `middleware.go` , aby som si bol isty ci mam riesi klienta alebo server)

Reply:
> toto si asi videl https://stackoverflow.com/a/45537161

To jsem neviděl. 🙂 Ale koukám že to píšou vesměs všude a odkazují se i na tuhle větu v RFC: 
![CleanShot 2022-11-23 at 20 48 57@2x](https://user-images.githubusercontent.com/215660/203634376-572f34cb-a950-40d9-86e0-450c9972549e.jpg)

Skoro bych z toho řekl že to použít ani nepůjde resp. neměli bysme, protože to hackujeme k něčemu k čemu to zřejmě není. 🙂 

Dumpnout ten request myslíš v `ContextMiddleware()`? V `RequestURI` to není.

![CleanShot 2022-11-23 at 20 57 53@2x](https://user-images.githubusercontent.com/215660/203636073-dc859922-3490-4968-91ce-1d41adf45591.jpg)

Reply:
jj, produkt to videl niekde v SNFK, tak nam to odporucil takto robit, ale je mozne ze to bola nejaka JS apka.

Reply:
@JakubMatejka tiez som to skusil, nefunguje to ani cez `curl`, zrusme tu `/#/` cast z definicie.

Reply:
aj keby sme to nejak hackli, tak to proste nebude fungovat vsade

Reply:
ok, taky bych řekl.

Reply:
https://github.com/keboola/keboola-as-code/pull/1001/commits/d9abaae94ce6c0ee06d13d581696d796c0d72a50